cmd/cloud-api-adaptor/main.go

```diff
@@ -35,10 +35,15 @@ type networkConfig struct {
 func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 
 	if len(os.Args) < 2 {
-		fmt.Printf("%s aws|azure|ibmcloud|libvirt|vsphere <options>\n", os.Args[0])
+		fmt.Printf("%s \n<aws|azure|ibmcloud|libvirt|version> [options]\n", os.Args[0])
```

Comment:
@Jordan9500 - did you mean to remove vsphere here?

Reply:
Good spot, I did not mean to remove that from the print. 
I will make another PR to re add that command to the print. 
Thank you :)